File: api-sample/oneec-csharp-sample/Oneec-Sample/models/MerchantCombinationProduct.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Oneec_Sample.models
{
    public class MerchantCombinationProduct
    {

        public String itemNumber;
        public List<MerchantCombinationInfo> combinationInfos;

        public String insertDt;
        public String modifiedDt;

        public void addCombinationInfo(MerchantCombinationInfo combinationInfo)
        {
            if (combinationInfos == null)
            {
                combinationInfos = new List<MerchantCombinationInfo>();
            }

            combinationInfos.Add(combinationInfo);
        }
    }
}

File: api-sample/oneec-java-sample/src/main/java/oneec/java/sample/models/MerchantCombinationProduct.java
package oneec.java.sample.models;

import java.math.BigDecimal;
import java.util.List;

public class MerchantCombinationProduct {
    public String itemNumber; // 貨號
    public List<MerchantCombinationInfo> combinationInfos; // 組合的內容資訊
    public String insertDt; // 格式 "yyyy-MM-ddTHH:mm:ss.SSSZ" ,UTC時間
    public String modifiedDt; // 格式 "yyyy-MM-ddTHH:mm:ss.SSSZ" ,UTC時間

    public static class MerchantCombinationInfo {
        public String productName; // 商品名稱
        public String itemNumber; // 貨號
        public int qty; // 數量
        public BigDecimal cost; // 成本
        public BigDecimal price; // 價格
    }
}
